packages/trpc/server/template-router/router.ts

```diff
@@ -64,11 +61,9 @@ export const templateRouter = router({
     .input(ZDuplicateTemplateMutationSchema)
     .mutation(async ({ input, ctx }) => {
       try {
-        const { templateId } = input;
-
         return await duplicateTemplate({
-          templateId,
           userId: ctx.user.id,
+          ...input,
```

Comment:
Why are we spreading like butter?

Reply:
I don't mind it but I also didn't mind the explicitness either so we don't accidentally pass new params that might not be needed as we extend our routers.

Reply:
In that case it wouldn't it be better to have zod be strict on the input?

Reply:
My point is more we might have a mutation/query take in something like `x, y, z` and then within that endpoint we have a method that takes `x, y` as arguments and then `z` for another methods argument.

Spreading is fine since this is mostly a straight passthru but explicit is also nice since it would mean we're never reverting back to explicit for the following kind of endpoints.

Reply:
Alright I'll revert

Hope you don't look at the teams router